at_functional_test/test/commons.dart

```diff
@@ -103,8 +103,8 @@ Future<String> read({int maxWaitMilliSeconds = 5000}) async {
       result = _queue.removeFirst();
       // result from another secondary is either data or a @<atSign>@ denoting complete
       // of the handshake
-      if (result.startsWith('data:') ||
-          (result.startsWith('error:')) ||
+      if (result.contains('data') ||
+          (result.contains('error')) ||
```

Comment:
Why do they no longer have the colons?

Reply:
Reverted the changes and now it is again data:/error: